.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Modify this '....'
+3. Execute this '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**OS and Polycube Version Details (please complete the following information):**
+- OS: [e.g. Ubuntu 18.04 LTS]
```

Comment:
Small alignment problem here.

Reply:
I would suggest the command to get OS and Polycube versions here, so that the user doesn't have to find them from himself.

Reply:
@mauriciovasquezbernal I found only one in the last line for version, been sorted. 
@frisso That's thoughtful! Also, it would be better to include commands/path for logs to collect.

Reply:
@mauriciovasquezbernal : could you please add directions (1) to get logs, (2) polycube version and (3) os version? Thank you very much.

Reply:
Sure, here we go:
kernel: `uname -r`
os: `cat /etc/*-release`
polycube: `polycubectl version`
logs: `cat /var/log/polycube/polycubed.log`

Reply:
added